junit-platform-suite-engine/src/test/java/org/junit/platform/suite/engine/testsuites/SuiteSuite.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.suite.engine.testsuites;
+
+import org.junit.platform.suite.api.SelectClasses;
+import org.junit.platform.suite.api.Suite;
+
+@Suite
+@SelectClasses(SelectClassesSuite.class)
+public class SuiteSuite {
+
+}
```

Comment:
We should use the `java-test-fixtures` Gradle plugin and move the two sample packages to the `testFixtures` source set like we do in `junit-vintage-engine`.

Reply:
Ah. This is pretty nice.

Resolved with 08feabb.